discord-scripts/fix-linear-embed.ts

```diff
@@ -132,32 +160,60 @@ async function processLinearEmbeds(
   logger: Log,
   linearClient: LinearClient,
 ) {
-  const issueUrlRegex =
-    /https:\/\/linear\.app\/([a-zA-Z0-9-]+)\/issue\/([a-zA-Z0-9-]+)(?:.*#comment-([a-zA-Z0-9]+))?/g
+  if (!issueTagRegex) {
+    logger.error("IssueTagRegex is not initialized.")
+    return
+  }
 
-  const matches = Array.from(message.matchAll(issueUrlRegex))
+  const urlMatches = Array.from(message.matchAll(issueUrlRegex))
+  const issueMatches = Array.from(message.matchAll(issueTagRegex))
 
-  if (matches.length === 0) {
+  if (urlMatches.length === 0 && issueMatches.length === 0) {
     return
   }
 
   const processedIssues = processedMessages.get(messageId) || new Set<string>()
   processedMessages.set(messageId, processedIssues)
 
-  const embedPromises = matches.map(async (match) => {
+  const uniqueMatches = new Set<string>()
+
+  urlMatches.forEach((match) => {
     const teamName = match[1]
```

Comment:
Note for all of these you can use [named capturing groups](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Named_capturing_group).